misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/CustomSystemClassLoader.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.runtime;
+
+import io.ballerina.projects.util.ProjectConstants;
+
+import java.io.*;
```

Comment:
It is a best practice to have each import separately than having them all together as `java.io.*`

Reply:
Done in [40229a4](https://github.com/ballerina-platform/ballerina-lang/pull/42183/commits/40229a40c6e8385fc2d56f004746b7a5494411f1) commit.